molecule/mysql_hardening/verify_tasks/service.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Load variables from role to use the "mysql_daemon" variable
+  ansible.builtin.include_role:
+    name: devsec.hardening.mysql_hardening
+    apply:
+      tags:
+        - never
```

Comment:
Awesome! I like it =)

Reply:
Yeah, I like that, too. Only works because we use the `always`-tasks.